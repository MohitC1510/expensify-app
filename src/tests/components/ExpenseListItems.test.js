import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expenses'
import ExpenseListItems from '../../components/ExpenseListItems';

test('should render expense', ()=>{
    const wrapper=shallow(<ExpenseListItems {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
})