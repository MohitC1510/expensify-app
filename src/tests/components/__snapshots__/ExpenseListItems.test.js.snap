// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense 1`] = `
<Link
  className="list-item"
  to="/edit/1"
>
  <div>
    <h3
      className="list-item__title"
    >
      gum
    </h3>
    <span
      className="list-item__subtitle"
    >
      Jan 1st, 1970
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $10.00
  </h3>
</Link>
`;
